csharp-basics/exercises/Collections/Exercise9/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Exercise9
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var randomList = new HashSet<string>() { "Apple", "Pineapple", "Cybertruck", "MacBook", "Android" };
+            Console.WriteLine("List with 5 items");
+            foreach(string thing in randomList)
+            {
+                Console.WriteLine(thing);
+            }
+            randomList.Clear();
+            randomList.Add("Mac and cheese");
+            randomList.Add("Mac and cheese");
+            Console.WriteLine("\nList with duplicate items");
```

Comment:
Vai šis atbilst noteikumam "Check if it is possible to add duplicated values" ?

Reply:
Tādā gadijumā, neesmu sapratis prasīto. HashSet fiziski nevar pievienot informācijas dublikātus.

Reply:
kā citādi bez testa pārbaudīt? 
domā vajag pārbaudi uz to ka Mac and cheese ir tikai viens sarakstā? 
vai kā? @ausmoons

Reply:
Es domāju par to, ka pats pievieno un tad izprintē pats message.